connman/plugins/wifi.c

```diff
@@ -1876,14 +1869,15 @@ static int wifi_scan(enum connman_service_type type,
 	if (wifi->tethering)
 		return -EBUSY;
 
-	if (type == CONNMAN_SERVICE_TYPE_P2P)
+	if (params->type == CONNMAN_SERVICE_TYPE_P2P)
```

Comment:
Can params itself be null?

Reply:
This is the upstream WiFi plugin we don't even build. But it cannot be `NULL` https://github.com/sailfishos/connman/pull/72/commits/e49a87134120b97b3f539e97d08fcdf71d2bb3ba